Client/Frontend/Browser/URLBarView.swift

```diff
@@ -652,7 +652,10 @@ extension URLBarView: TabLocationViewDelegate {
     }
 
     func tabLocationViewDidTapLocation(tabLocationView: TabLocationView) {
-        let locationText = delegate?.urlBarDisplayTextForURL(locationView.url)
+        var locationText = delegate?.urlBarDisplayTextForURL(locationView.url)
+        if let utf8Host = locationView.url?.host?.asciiHostToUTF8() {
```

Comment:
Same here -- extract `locationView.url?.host?` using `if let` so that you don't need the force unwrapping below.